gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -86,7 +86,11 @@ function GM:PlayerBindPress(ply, bind, pressed)
 
 			return true
 		elseif TBHUD:PlayerIsFocused() then
-			return TBHUD:UseFocused()
+			if input.IsButtonDown(KEY_LALT) then
+				return TBHUD:ToggleFocused(input.IsButtonDown(KEY_LSHIFT))
```

Comment:
maybe add some small comments to these alternatives to quickly give an overview on their functionality